helm/opendistro-es/values.yaml

```diff
@@ -11,6 +11,59 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
+certmanager:
+  enabled: false
+
+  ca:
+    commonName: example.com
+    duration: 21600h
+    renewBefore: 360h
+    keySize: 2048
+
+  elasticsearch:
+    rest:
+      enabled: true
+      secretName: es-rest
+      commonName: example.com
+      duration: 21600h
+      renewBefore: 360h
+      keySize: 2048
+      usages:
+      - server auth
+      - client auth
+    transport:
+      enabled: true
+      secretName: es-transport
```

Comment:
I believe these secret names are going to be used for the different secrets under `elasticsearch:` and `kibana:`. Maybe also add them there.  
`elasticsearch:
   ssl:
      transport:
        existingCertSecret: **es-transport**
`?

Reply:
Not really sure i see the problem here. Can't kibana just mount the transport certificate? 
```
kibana
  ssl:
    elasticsearch:
      enabled: true
      existingCertSecret: es transport
      existingCertSecretCertSubPath: elk-rest-crt.pem
      existingCertSecretKeySubPath: elk-rest-key.pem
      existingCertSecretRootCASubPath: elk-rest-root-ca.pem
```

Reply:
There is no issue at all. What I was asking is that the secret names are not added there and would be nice to have them. for example currently the secret name for `existingCertSecret` is not specified in values.yaml. And I was suggesting to add it (also for the others)
```
kibana
  ssl:
    elasticsearch:
      enabled: false
      existingCertSecret: 
      existingCertSecretCertSubPath: elk-rest-crt.pem
      existingCertSecretKeySubPath: elk-rest-key.pem
      existingCertSecretRootCASubPath: elk-rest-root-ca.pem
```

Reply:
So, you mean that we should not set any default names for the certificates, whereby they must be overridden by a user?

Reply:
that or put the secret names in their respective configurations. e.g.,

```
kibana
  ssl:
    elasticsearch:
      enabled: false
      existingCertSecret: **es-rest**
      existingCertSecretCertSubPath: elk-rest-crt.pem
      existingCertSecretKeySubPath: elk-rest-key.pem
      existingCertSecretRootCASubPath: elk-rest-root-ca.pem
```